tools/run_cargo_fmt.sh

```diff
@@ -44,24 +44,19 @@ if git status --porcelain | grep '^.M.*\.rs' -q; then
 	fi
 fi
 
-set +e
-let FAIL=0
-set -e
-
-# Find folders with Cargo.toml files in them and run `cargo fmt`.
 if [ "$1" == "diff" ]; then
-	# Just print out diffs and count errors, used by Travis
+	# Just print out diffs, used by Travis.
 	CARGO_FMT_ARGS="-- --check"
 fi
-for f in $(find . | grep Cargo.toml); do
-	pushd $(dirname $f) > /dev/null
-	cargo-fmt $CARGO_FMT_ARGS || let FAIL=FAIL+1
-	popd > /dev/null
-done
```

Comment:
The `tools/` are currently not in the workspace, so this change will prevent them from being checked for formatting. In principle, they could be added to the workspace as well to simplify things, except that the following configuration (which isn't relevant for these tools) is defined for the whole workspace - due to Cargo's rules.

https://github.com/tock/tock/blob/927cb371dad818cae861cd742f203a40dfcbce1c/Cargo.toml#L51-L61

An alternative would be to have the main workspace for boards, and a second workspace in the `tools/` folder for tools.